pyproject.toml

```diff
@@ -26,6 +26,7 @@ classifiers = [
 dependencies = [
     "nomad-lab>=1.2.2.dev399",
     "xmltodict==0.13.0",
+    "fairmat-readers-xrd~=0.0.3"
```

Comment:
Why the approx sign?

Reply:
this way we can specify to use a compatible version with 0.0.3, which will be the latest version of the form: `0.0.*`. Ideally, the newer version of this form should be backward compatible.
https://peps.python.org/pep-0440/#compatible-release

Reply:
Ah, cool I didn't know this. Thanks for the info :)